mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tasks/SynchronizeStatusesOnKafka.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.mulighetsrommet.api.tasks
+
+import com.github.kagkarlsson.scheduler.task.helper.RecurringTask
+import com.github.kagkarlsson.scheduler.task.helper.Tasks
+import com.github.kagkarlsson.scheduler.task.schedule.Daily
+import kotlinx.coroutines.runBlocking
+import no.nav.mulighetsrommet.api.services.KafkaSyncService
+import no.nav.mulighetsrommet.slack_notifier.SlackNotifier
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.ZoneId
+
+class SynchronizeStatusesOnKafka(kafkaSyncService: KafkaSyncService, slackNotifier: SlackNotifier) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    val task: RecurringTask<Void> = Tasks
+        .recurring("synchronize-statuses-kafka", Daily(LocalTime.MIDNIGHT))
```

Comment:
Er tanken at denne jobben skal være felles for evt. andre statuser for andre modeller som skal synces til kafka også (f.eks. tiltakstyper, om det blir aktuelt), eller vil vi opprette en egen jobb for dette om det blir relevant?
Tenker bare om navnet burde inneholde `tiltaksgjennomføringer` også, eller om tanken er "en jobb for alt".